keep/api/utils/import_ee.py

```diff
@@ -0,0 +1,16 @@
+import os 
+import sys
+import pathlib
+
+ee_enabled = os.environ.get("EE_ENABLED", "false") == "true"
+if ee_enabled:
+    path_with_ee = (
+        str(pathlib.Path(__file__).parent.resolve()) + "/../../../ee/experimental"
+    )
```

Comment:
this will probably won't work in different deployment types (e.g., docker?)

Reply:
Goooooood catch!

Reply:
Resolved <3